src/python/dxpy/scripts/dx.py

```diff
@@ -2834,12 +2834,24 @@ def handle_arg_conflicts(args):
         if args.repository and args.remote:
             build_parser.error("Nextflow pipeline built from a remote Git repository is always built using the Nextflow Pipeline Importer app. This is not compatible with --remote.")
 
+        if args.cache_docker and args.remote:
+            build_parser.error("Nextflow pipeline built with an option to cache the docker images is always built using the Nextflow Pipeline Importer app. This is not compatible with --remote.")
+
         if args.git_credentials and not args.repository:
             build_parser.error("Git credentials can be supplied only when building Nextflow pipeline from a Git repository.")
 
         if args.nextflow and args.mode == "app":
             build_parser.error("Building Nextflow apps is not supported. Build applet instead.")
 
+        if args.cache_docker:
```

Comment:
check `if args.cache_docker and not args.nextflow:` as in line 2831?

Reply:
thanks!
[888fcd1](https://github.com/dnanexus/dx-toolkit/pull/1280/commits/888fcd10eaa623dd0c6ae9ac1aceb85c2bf17aa1)